WeatherApp/ViewControllers/WeatherListViewController.swift

```diff
@@ -7,14 +7,19 @@
 //
 
 import UIKit
+import RxSwift
+import RxCocoa
 import PureLayout
 
 class WeatherListViewController: UIViewController {
     
     private let weatherListView = WeatherListView()
```

Comment:
Why not just add UITableView directly to the viewController? No need for wrapping it inside a view. 
It isn't really a part of this PR, but I didn't pay attention to it before.